### PR TITLE
Tracker disarmed pwm bug fix

### PR DESCRIPTION
**Pull request by IamPete1**
This should fix https://github.com/ArduPilot/ardupilot/issues/11575

### CODE REVIEW

File: AntennaTracker/tracking.cpp [hunk 1]
```diff
@@ -105,31 +105,44 @@ void Tracker::update_tracking(void)
...
-    case AUTO:
-        if (vehicle.location_valid) {
-            update_auto();
-        } else if (tracker.target_set) {
-            update_scan();
+        switch (g.disarm_pwm) {
+        case pwm_disarmed::ZERO_PWM:
+            SRV_Channels::set_output_pwm(SRV_Channel::k_tracker_yaw, 0);
+            SRV_Channels::set_output_pwm(SRV_Channel::k_tracker_pitch, 0);
+
```
Comment (IamPete1) [inline AntennaTracker/tracking.cpp]: this switch statement falls thrgouht, should also add defautl case

File: AntennaTracker/defines.h [hunk 2]
```diff
@@ -31,6 +31,11 @@ enum mode_reason_t {
     MODE_REASON_GCS_COMMAND,
 };
 
+enum pwm_disarmed {
```
[the remark this answers is not in the document]
Comment (IamPete1) [reply AntennaTracker/defines.h]: fixed

File: AntennaTracker/tracking.cpp (hunk 1 above)
Comment (IamPete1) [reply AntennaTracker/tracking.cpp]: fixed

File: AntennaTracker/defines.h [hunk 3]
```diff
@@ -31,6 +31,11 @@ enum mode_reason_t {
...
+    ZERO_PWM = 0,
```
[the remark this answers is not in the document]
Comment (IamPete1) [reply AntennaTracker/defines.h]: fixed

**Comment by IamPete1** [comment]
Have finally got round to fixing and checking this, it adds a new param the does nothing new by default. This outputs no/zero pwm when disarmed. It gives the option to set to two and output trim pwm. 

Both have been tested on my gimbal come antenna tracker

**Comment by rmackay9** [comment]
We have the equivalent parameter in both Copter and Rover and it's called MOT_SAFE_DISARM.  We don't really have a motors library in tracker but I wonder if we could just call it SAFE_DISARM?

File: AntennaTracker/defines.h [hunk 4]
```diff
@@ -31,6 +31,11 @@ enum mode_reason_t {
     MODE_REASON_GCS_COMMAND,
 };
 
+enum class PAWDisarmed {
```
Comment (rmackay9) [inline AntennaTracker/defines.h]: typo here maybe in the class name?

**Comment by rmackay9** [comment]
This looks good to me besides the couple of questions above re naming.

File: AntennaTracker/defines.h (hunk 4 above)
Comment (IamPete1) [reply AntennaTracker/defines.h]: fixed

**Comment by IamPete1** [comment]
typo corrected and I have changed the param to 'SAFE_DISARM_PWM' but happy to change to something else if anyone has any strong feelings

**Comment by rmackay9** [comment]
Merged, thanks!

**Comment by IamPete1** [comment]
@rmackay9 @peterbarker only just occurred to me, I guess I should back port this to the beta branch?

**Comment by peterbarker** [comment]
On Fri, 19 Jul 2019, Peter Hall wrote:

> @rmackay9 @peterbarker only just occurred to me, I guess I should back port this to the beta branch?

Yes.

**Pull request merged**